### PR TITLE
Add mqtt support to onewire

### PR DESCRIPTION
**Pull request by AnDann**

### CODE REVIEW

File: hardware/onewire/onewire.h [hunk 1]
```diff
@@ -69,6 +69,12 @@
 #define OW_DEBUG_POLL(...)    ((void) 0)
 #endif
 
+#ifdef DEBUG_ECMD_OW_GET
+#include "core/debug.h"
+#define OW_DEBUG_GET(str...) debug_printf ("OW-GET: " str)
+#else
+#define OW_DEBUG_GET(...)    ((void) 0)
```
Comment (eku) [inline hardware/onewire/onewire.h]: ???

File: hardware/onewire/onewire_mqtt.c [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+#endif
+  }
+  if(++ow_act >= OW_SENSORS_COUNT)
+  {
+    ow_act=0;
+    ow_mqtt_new = 0;
+  }
+}
+
+static mqtt_callback_config_t mqtt_callback_config = {
```
Comment (eku) [inline hardware/onewire/onewire_mqtt.c]: Does this structure realy have to be in RAM or can we modify the MQTT coude that is using it to acess PROGMEM?

Comment (AnDann) [inline hardware/onewire/onewire_mqtt.c]: You're right the callback´s and connection struct should not change at runtime. I will look into this.

File: hardware/onewire/onewire.h (hunk 1 above)
Comment (AnDann) [inline hardware/onewire/onewire.h]: Oh there were some debug_printf´s in the ecmd code which spamed my syslog should i create a seperate PR for this?

**Comment by eku** [comment]
Please rework this PR as it is no longer up-to-date with the MQTT code. Thanks.

**Pull request closed without merging**